translate/src/modules/entitydetails/components/EntityDetails.tsx

```diff
@@ -88,21 +93,35 @@ export function EntityDetails(): React.ReactElement<'section'> | null {
     [dispatch],
   );
 
+  const { code } = useContext(Locale);
+
+  const openTeamComments = useCallback(() => {
+    const teamCommentsTab = commentTabRef.current;
+    const index = teamCommentsTab?._reactInternalFiber.index ?? 0;
```

Comment:
We should properly fix this at some point. But for now let's at least highlight it a bit better in the code.

```suggestion
    // FIXME: This is an ugly hack.
    // https://github.com/mozilla/pontoon/issues/2300
    const index = teamCommentsTab?._reactInternalFiber.index ?? 0;
```